pkg/controller/metrics/metrics.go

```diff
@@ -597,6 +594,6 @@ func GetClusterDeploymentType(obj metav1.Object) string {
 // optionally an outcome in a consistent fashion.
 func ObserveControllerReconcileTime(controller string, start time.Time, outcome ReconcileOutcome, logger log.FieldLogger) {
 	dur := time.Since(start)
-	metricControllerReconcileTime.WithLabelValues(controller).Observe(dur.Seconds())
-	logger.WithFields(log.Fields{"elapsed": dur, "outcome": outcome}).Info("reconcile complete")
+	metricControllerReconcileTime.WithLabelValues(controller, string(outcome)).Observe(dur.Seconds())
```

Comment:
this change should be in the previous commit

Reply:
I am cursing you for noticing that. :)